common/types/config.ts

```diff
@@ -25,6 +25,9 @@ export const configSchema = schema.object({
   smartAnomalyDetector: schema.object({
     enabled: schema.boolean({ defaultValue: false }),
   }),
+  branding: schema.object({
+    label: schema.string({ defaultValue: 'AI assistant' }),
+  }),
 });
```

Comment:
nit: do we need i18n here?

Reply:
This value will be from config yaml, I'm not sure if i18n will even work for config yaml

Reply:
You mean i18n for the default value?

Reply:
> This value will be from config yaml, I'm not sure if i18n will even work for config yaml

I see, if the value from config yaml, the value should be whatever specified in yaml file. can we make default value as empty? that will use the fallback value that supports i18n

Reply:
Good point, I will update